### PR TITLE
upgrade JSONField for form_processor

### PR DESCRIPTION
**Pull request by nickpell**
upgrading our third party JSONField library from 0.5.7 to 1.0.3 (same workflow as https://github.com/dimagi/commcare-hq/pull/10432)

### CODE REVIEW

**Comment by nickpell** [comment]
@snopoke cc: @kaapstorm

File: corehq/form_processor/models.py [hunk 1]
```diff
@@ -199,8 +199,8 @@ class XFormInstanceSQL(DisabledDbMixin, models.Model, RedisLockableMixIn, Attach
     # The time at which the server has received the form
     received_on = models.DateTimeField()
 
-    auth_context = JSONField(lazy=True, default=dict)
-    openrosa_headers = JSONField(lazy=True, default=dict)
+    auth_context = JSONField(default=dict)
+    openrosa_headers = JSONField(default=dict)
```
Comment (snopoke) [inline corehq/form_processor/models.py]: Does the new one not support lazy loading?

**Comment by snopoke** [comment]
:+1: with minor question

File: corehq/form_processor/models.py (hunk 1 above)
Comment (nickpell) [inline corehq/form_processor/models.py]: It looks like the new version does lazy loading by default.
Old:
https://github.com/derek-schaefer/django-json-field/blob/master/json_field/fields.py#L118
New:
https://github.com/bradjasper/django-jsonfield/blob/master/jsonfield/fields.py#L69

Comment (snopoke) [inline corehq/form_processor/models.py]: 👍

**Pull request merged**